packages/components/src/components/input/input.ts

```diff
@@ -89,39 +96,40 @@ export default class SdInput extends SolidElement implements SolidFormControl {
     | 'time'
     | 'url' = 'text';
 
-  /** The name of the input, submitted as a name/value pair with form data. */
-  @property() name = '';
+  /** The input's size. */
+  @property({ reflect: true }) size: 'lg' | 'md' | 'sm' = 'lg';
+
+  /**
+   * Tells the browser what type of data will be entered by the user, allowing it to display the appropriate virtual
+   * keyboard on supportive devices.
+   */
+  @property() inputmode: 'none' | 'text' | 'decimal' | 'numeric' | 'tel' | 'search' | 'email' | 'url';
 
+  // --------------------------- GENERIC --------------------------- //
```

Comment:
Do you mean by "Generic" that these are things which are inherited by the original input? :) Not sure if I like it, but don't have strong feelings on it. If you leave it, maybe we can have a better wording like "original input attributes" or similar.

Reply:
it was a label for me to separate the props categorically while developing, will remove it!